sys-cluster/crmsh/files/crmsh-4.4.0-utils-wait4dc.patch

```diff
@@ -0,0 +1,413 @@
+--- a/crmsh/utils.py	2022-02-17 05:38:34.000000000 +0100
```

Comment:
Above the patch you maybe want to add a short explaination like mentioned here:

https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/#clean-patch-howto

Reply:
i wanted to separate this, but i did the pull request here on github and found no way to keep parallax separated. also don't know how this signing works, as my "Bug" link is not parsed as well. so i thought "Signed-off-by: Full Name <email@example.com>" also will not work. 
any idea how to remove parallax out of this request?

Reply:
yes, crmsh does not work if parallax is compiled for python 3.9

Reply:
Then keep it in this PR, just make it a proper revbump, so update the EAPI while you are at it and also the year in the title.

Reply:
thanks for your guideance... what is needed for a proper revbump? changed file name, date and EAPI

Reply:
you mean make a new pull request for parallax only?

Reply:
added you as collaborator if you want to add your "senf" ;)

Reply:
> you mean make a new pull request for parallax only?

no, use this PR, squash all commits regarding parallax into one commit, then squash all commits regarding crmsh into another commit